packages/config/config/devices/0x0109/zl7435-5.json

```diff
@@ -0,0 +1,48 @@
+// Vision Security ZL7435-5
+// In-wall Switch (2 Relays)
+{
+	"manufacturer": "Vision Security",
+	"manufacturerId": "0x0109",
+	"label": "ZL7435-5",
+	"description": "In-wall Switch Module (2 Relays)",
+	"devices": [
+		{
+			"productType": "0x2017",
+			"productId": "0x1719",
+			"zwaveAllianceId": 1367
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Button Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Switch (default)",
```

Comment:
We render a default label in the UI based on the defaultValue field, so please remove any (default) from any labels, descriptions, etc.